mocks/transaction.js

```diff
@@ -0,0 +1,55 @@
+const mockGetAll = jest.fn();
+const mockGetAllTransaction = jest.fn();
+const mockGetTransaction = jest.fn();
+const mockSetTransaction = jest.fn();
+const mockUpdateTransaction = jest.fn();
+const mockDeleteTransaction = jest.fn();
+
+class Transaction {
+  getAll(...refsOrReadOptions) {
+    mockGetAll(...arguments);
+    mockGetAllTransaction(...arguments);
+    // TODO: Assert that read options, if provided, are the last argument
```

Comment:
Want to add this as an issue so we don't lost track of it?

Reply:
https://github.com/Upstatement/firestore-jest-mock/issues/47